src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -370,6 +369,19 @@ export class SkyDatepickerInputDirective
     this.datepickerComponent.disabled = disabled;
   }
 
+  /**
+   * Detects changes to the underlying input element's value and updates the ngModel accordingly.
+   * This is useful if you need to update the ngModel value before the input element loses focus.
+   */
+  public detectInputValueChange() {
```

Comment:
I know our source code is inconsistent, but we've been trying to enforce some code style rules for all future pull requests. 

For the `detectInputValueChange` and `onValueChange` methods, we should set the return type, even if it's `void`. See: https://docs.blackbaud.com/skyux-team-docs/style#methods